republication-tracker-tool.php

```diff
@@ -191,15 +195,50 @@ public function enable_pixel_query_vars($vars){
 		return $vars;
 
 	}
+
+	/**
+	 * Create tracking pixel HTML markup.
+	 *
+	 * @param $post_id Id of the post to track.
+	 */
+	public static function create_tracking_pixel_markup( $post_id ) {
+		$analytics_id = get_option( 'republication_tracker_tool_analytics_id' );
+		return sprintf(
+			// %1$s is the javascript source, %2$s is the post ID, %3$s is the plugins URL
+			'<img id="republication-tracker-tool-source" src="%1$s/?republication-pixel=true&post=%2$s&ga=%3$s" style="width:1px;height:1px;">',
+			esc_attr( get_site_url() ),
+			esc_attr( $post_id ),
+			esc_attr( $analytics_id )
+		);
+	}
+
+	/**
+	 * Get attribution text, which will be inserted at the end of the copyable content.
+	 *
+	 * @param $post_permalink Permalink of the shared post.
+	 */
+	public static function create_attribution_markup( $post_permalink = '' ) {
+		$display_attribution = get_option( 'republication_tracker_tool_display_attribution', 'on' );
```

Comment:
Why not a true boolean value, [like here](https://github.com/Automattic/republication-tracker-tool/pull/103/files#diff-ffe4c5eae4f71cfc06cca2fc4501f5eac2646795e3253f4b5d45c3efe226b592R132)?

Reply:
WP's settings are saving a checkbox input value as a string (empty or `'on'`). Unified to avoid confusion in 77d2e879a9d14035cccc2d8517a331df954b2fbe